uisdk/src/main/res/values-nl/strings.xml

```diff
@@ -262,10 +262,10 @@
     <string name="kh_uisdk_permissions">RECHTEN</string>
     <!-- Profile -->
     <string name="kh_uisdk_phone_number_hint">Telefoonnummer</string>
-    <string name="kh_uisdk_pickup_type_curbside">stoeprand</string>
+    <string name="kh_uisdk_pickup_type_curbside">Curbside</string>
     <string name="kh_uisdk_pickup_type_meet_and_greet">Ontmoet en Groet</string>
     <string name="kh_uisdk_pickup_type_standby">Stand-by</string>
-    <string name="kh_uisdk_prebook_timezone_title">De boeking wordt in lokale tijd gemaakt</string>
+    <string name="kh_uisdk_prebook_timezone_title">De reservering wordt gemaakt in de lokale tijd van ophalen ( %1$s )</string>
```

Comment:
The placeholder at the end of this string, does not seem to be consistent with other locales